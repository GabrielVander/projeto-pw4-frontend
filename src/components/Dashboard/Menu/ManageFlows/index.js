import React from 'react';
import Card from 'react-bootstrap/Card';
import {BsGrid1X2} from 'react-icons/bs';
import './styles.css';
import { motion } from 'framer-motion';
import {useHistory} from 'react-router-dom';

function ManageFlows() {
	const history = useHistory();

	return (
		<>
			<motion.div
				whileHover={{ scale: 1.1 }}
				whileTap={{ scale: 0.9 }}
			>
				<Card bg='primary' onClick={() => history.push('/flows')}>
					<Card.Header>Manage flows</Card.Header>
					<Card.Body>
						<BsGrid1X2 size={50} />
					</Card.Body>
				</Card>
			</motion.div>
		</>
	);
}

export default ManageFlows;