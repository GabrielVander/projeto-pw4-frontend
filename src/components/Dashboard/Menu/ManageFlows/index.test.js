import React from 'react';
import { render } from '@testing-library/react';
import ManageFlows from './index';

describe('Component renders properly', () => {
	test('Matches snapshot', () => {
		const component = render(<ManageFlows/>);

		expect(component).toBeDefined();
		expect(component).toMatchSnapshot();
	});
});