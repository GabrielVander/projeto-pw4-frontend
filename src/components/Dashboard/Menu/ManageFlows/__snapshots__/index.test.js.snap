// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders properly Matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="card bg-primary"
        >
          <div
            class="card-header"
          >
            Manage flows
          </div>
          <div
            class="card-body"
          >
            <svg
              fill="currentColor"
              height="50"
              size="50"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="50"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M6 1H1v14h5V1zm9 0h-5v5h5V1zm0 9h-5v5h5v-5zM0 1a1 1 0 011-1h5a1 1 0 011 1v14a1 1 0 01-1 1H1a1 1 0 01-1-1V1zm9 0a1 1 0 011-1h5a1 1 0 011 1v5a1 1 0 01-1 1h-5a1 1 0 01-1-1V1zm1 8a1 1 0 00-1 1v5a1 1 0 001 1h5a1 1 0 001-1v-5a1 1 0 00-1-1h-5z"
                fill-rule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="card bg-primary"
      >
        <div
          class="card-header"
        >
          Manage flows
        </div>
        <div
          class="card-body"
        >
          <svg
            fill="currentColor"
            height="50"
            size="50"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="50"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M6 1H1v14h5V1zm9 0h-5v5h5V1zm0 9h-5v5h5v-5zM0 1a1 1 0 011-1h5a1 1 0 011 1v14a1 1 0 01-1 1H1a1 1 0 01-1-1V1zm9 0a1 1 0 011-1h5a1 1 0 011 1v5a1 1 0 01-1 1h-5a1 1 0 01-1-1V1zm1 8a1 1 0 00-1 1v5a1 1 0 001 1h5a1 1 0 001-1v-5a1 1 0 00-1-1h-5z"
              fill-rule="evenodd"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
